content/300-guides/200-deployment/001-deploying-prisma.mdx

```diff
@@ -19,16 +19,22 @@ Moreover, the user traffic pattern of your application is also an important fact
 
 ### Long-running process (PaaS)
 
-Your Node.js process is continuously running and handles multiple requests. Your application can be deployed to a Platform-as-a-Service like [Heroku](./deployment-guides/deploying-to-heroku), [Koyeb](./deployment-guides/deploying-to-koyeb), as a Docker container to Kubernetes, or as a Node.js process on a virtual machine.
+Your application is [traditionally deployed](./traditional-deployments) if a Node.js process is continuously running and handles multiple requests. Your application can be deployed to a Platform-as-a-Service like [Heroku](./traditional-deployments/deploying-to-heroku), [Koyeb](./traditional-deployments/deploying-to-koyeb), as a Docker container to Kubernetes, or as a Node.js process on a virtual machine.
 
 See also: [Connection management in long-running processes](/guides/performance-and-optimization/connection-management#long-running-processes)
 
 ### Serverless (FaaS)
 
-Node.js processes of your application (or subsets of it broken into functions) are started as requests come in.
+Your application is [serverless](./serverless-deployments) if the Node.js processes of your application (or subsets of it broken into functions) are started as requests come in.
 
 Serverless environments have the concept of warm starts, which means that for subsequent invocations of the same function, it may use an already existing container that has the allocated processes, memory, file system (`/tmp` is writable on AWS Lambda), and even DB connection still available.
 
 Typically, any piece of code [outside the handler](https://docs.aws.amazon.com/lambda/latest/dg/gettingstarted-features.html#gettingstarted-features-programmingmodel) remains initialized.
 
 See also: [Connection management in serverless environments](/guides/performance-and-optimization/connection-management#serverless-environments-faas)
+
+### Edge (Distributed FaaS)
+
+Your application is [edge deployed](./edge-deployments) if your application is [serverless](#serverless-faas) and the functions are distributed across one or more regions.
+
+Typically, edge environments also have a different runtime than a standard or serverless environment, leading to common APIs being unavailable.
```

Comment:
Note: Here you use "standard" instead of "traditional". Intentional?

Reply:
Semi-intentional. I'm still not convinced on terminology. "Traditional"? "Standard"? "Long-running"???

Reply:
**Long-running** servers are the **traditional** way of running Node.js applications, which used to be **standard**. Today one could argue that in some segments more short-lived serverless functions have taken over that role already.

(scnr)